Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -90,6 +90,38 @@
           "required": false,
           "groupName": "advanced",
           "helpMarkDown": "Minimatch pattern to filter files to be downloaded."
```

Comment:
don't you need to bump up task version?

Reply:
Was thinking, if we can get this today, we don't need to bump up as Omesh had already bumped up the version. But bumped up anyway just not to forget if this is delayed.